gap/Magic.gd

```diff
@@ -303,6 +303,31 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!         <Mark><A>gap_root_relative_path</A></Mark>
+#!         <Item>
+#!             Either a boolean, or a string containing a relative path.
+#!             By default (if this option is not set, or is set to <K>false</K>),
+#!             references in the generated documentation referring to external documentation
+#!             (such as the GAP manual) are encoded using absolute paths.
+#!             This is fine as long as the documentation stays on only a single
+#!             computer, but is problematic when preparing documentation that should be
+#!             used on multiple computers, e.g., when creating a distribution archive of
+#!             a GAP package.<Br/>
+#!             Thus, if a relative path is provided via this option (or if it is set to true,
+#!             in which case the relative path <F>../../..</F> is used), then &AutoDoc;
+#!             and &GAPDoc; attempt to replace all absolute paths in references to GAP
+#!             manuals by paths based on this relative path.<P/>
+#!             
+#!             On a technical level, &AutoDoc; passes the relative path to the
+#!             <A>gaproot</A> parameter of <Ref Func='MakeGAPDocDoc'
+#!             BookName='gapdoc'/><P/>
+#             
+#             Additionally, you can also set the global option <C>relativePath</C>
+#             to a boolean or a string with the same effect (for details on global options,
+#             see <Ref Chap="Options Stack" BookName="ref"/>). If both this option
+#             record entry and the global option are set, then the global option takes
+#             precedence.
+#!         </Item>
```

Comment:
This is weird. To a casual observer, it seems as if the `!` is left out by accident. I'd really just remove this whole paragraph.